AstarteDeviceSDKCSharp/Transport/AstarteTransportFactory.cs

```diff
@@ -47,5 +49,18 @@ internal class AstarteTransportFactory
                     return null;
             }
         }
+
+        public static AstarteTransport? CreateAstarteTransportOfflineAstarteTransport
+        (AstarteProtocolType protocolType, string astarteRealm,
+        string deviceId, dynamic protocolData, AstarteCryptoStore astarteCryptoStore,
+        TimeSpan timeOut, AstarteFailedMessageStorage astarteFailedMessageStorage)
+        {
+            return new AstarteTransportOffline(new MutualSSLAuthenticationMqttConnectionInfo(
+                new Uri("mqtts://broker.astarte.localhost:8883"),
```

Comment:
The offline transport should not have a MutualSSLAuthenticationMqttConnectionInfo and Uri.
What about creating a specific IMqttConnectionInfo such as OfflineMqttConnectionInfo or NoMqttConnectionInfo ? Also should another constructor be added to `AstarteMqttV1Transport` that accepts `OfflineMqttConnectionInfo`.